charts/openstack-cluster/templates/_helpers.tpl

```diff
@@ -99,6 +99,14 @@ mirrors and additional packages.
 {{- end }}
 {{- if or $registryMirrors $files }}
```

Comment:
@m-bull 

We want this file to always be created, but it is inside an `if` statement. You need to do some shuffling.